OpenTelemetry.sln

```diff
@@ -198,6 +198,8 @@ Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Examples.GrpcService", "exa
 EndProject
 Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "getting-started", "docs\logs\getting-started\getting-started.csproj", "{B3F03725-23A0-4582-9526-F6A7E38F35CC}"
 EndProject
+Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "OpenTelemetry.Exporter.InMemory", "src\OpenTelemetry.Exporter.InMemory\OpenTelemetry.Exporter.InMemory.csproj", "{9BCEA68B-50E2-4A3A-93E6-B51AF612BCC1}"
```

Comment:
Sorry for missing this. I was actually preparing the PR this noon but got interrupted with other stuff. (and I also missed the top level README file, will add it today).

Reply:
#1340

Reply:
No prob, it was easy enough to fix up. We could throw a vs-build task into our windows-CI to catch stuff like this?